squirrel.worker/src/main/java/org/dice_research/squirrel/metadata/CrawlingActivity.java

```diff
@@ -123,6 +123,7 @@ public Model prepareMetadataModel() {
         model.add(activity, PROV_O.startedAtTime, model.createTypedLiteral(dateStarted));
         model.add(activity, PROV_O.endedAtTime, model.createTypedLiteral(dateEnded));
         model.add(activity, Squirrel.approxNumberOfTriples, model.createTypedLiteral(numberOfTriples));
+        model.add(activity, Squirrel.depth, model.createTypedLiteral(Integer.parseInt(uri.getData(Constants.URI_DEPTH).toString())));
```

Comment:
First, it should be checked whether the URI has a depth value.

Reply:
you are right, forgot about it